docs/loader.md

```diff
@@ -0,0 +1,9 @@
+---
+short-description: Loader and Initialization
```

Comment:
You know that you could have comment in the `.h` file instead of standalone `.md` files?

Something like:

``` c
/**
 * SECTION: loader
 * symbols:
 * - wpe_loader_interface
 * - wpe_loader_init
 * - wpe_loader_get_loaded_implementation_library_name
 */
```

(listing symbols is not mandatory for symbol that are present in the current `.h` file (or its corresponding `.c`)).

Reply:
This is definitely a nice thing to know—I will rework this PR to keep the `--c-smart-index` option and add `SECTION: …` comments where needed. On the other hand, I still want to keep the top-level Markdown document, and filling in there some general information about the library at some point. An additional Markdown page with an overview of how to implement a new WPE backend would be a great addition, too.